src/ApiClient/ServerClient.ts

```diff
@@ -303,6 +292,17 @@ export default new (class ServerClient extends ApiClient<IEvents> {
                 Api: `Tgstation.Server.Api/` + API_VERSION,
                 "Webpanel-Version": VERSION
             },
+            securityWorker: async () => {
+                const tok = await this.wait4Token();
+                if (!tok) {
+                    return; // undefined is valid. Also it means that we logged out
+                }
```

Comment:
This may break shit, the app was made with the expectation that if a request was made while unauthenticated, it would be paused until the user was authenticated

Reply:
The 401 error handler @ `apiResponseInterceptor`

Reply:
I see, can you add a comment to say that the retry will get handled in the response interceptor if the token is missing?

Reply:
Is there a way to avoid having to send unauthenticated requests tho?